bigcache.go

```diff
@@ -104,21 +104,21 @@ func (c *BigCache) Close() error {
 // Get reads entry for the key.
 // It returns an ErrEntryNotFound when
 // no entry exists for the given key.
-func (c *BigCache) Get(key string) ([]byte, error) {
+func (c *BigCache) Get(key []byte) ([]byte, error) {
```

Comment:
Breaking changes

Reply:
Yes I know... ok... first time user of github here.

1. I changed fnv.go and submitted a pull request here.
2. basically only need to change fnv.go (this is the only file i'm changing) for the pull request

I thought that ends there for the pull while I carry on with a bigcache variant that I thought will optimize "key" by converting it to []byte for my own use

3. I didn't realise the Pull Request is still relevant after I changed the rest. Can we just focus on fnv.go? I didn't know that PR applies while I'm editing the other files in master.

Reply:
Just rebase that changes.